docker/README.md

```diff
@@ -107,8 +107,10 @@ docker run --rm -it ... matrixdotorg/sytest-synapse:buster tests/20profile-event
 
 ## Building the containers
 
-The containers are built by executing `./build.sh`. You will then have to push
-them up to Docker Hub with `./push.sh`.
+These are automatically build and deployed on the `develop` branch by GitHub Actions.
+See `.github/workflows.docker.yml` for the gory details.
+
+Historically, the containers were built by executing `./build.sh`, then pushed up to Docker Hub with `./push.sh`.
```

Comment:
Perhaps we just want to remove these lines? If someone needs to know how things were done in a previous life, git blame can bring them back to the history — it *feels like* it makes sense for these kinds of living documents to focus on how things are done *now* (as much as historical context is nice sometimes).

Reply:
It was useful for me in #1133 to be able to build these containers. (I'm not able to run sytest directly from my checkout, due to #1114). But I don't think that's a very persuasive argument for keeping `build.sh` around.